pkg/scheduler/apis/config/v1beta2/defaults.go

```diff
@@ -55,14 +53,14 @@ func SetDefaults_KubeSchedulerConfiguration(obj *v1beta2.KubeSchedulerConfigurat
 	// Only apply a default scheduler name when there is a single profile.
 	// Validation will ensure that every profile has a non-empty unique name.
 	if len(obj.Profiles) == 1 && obj.Profiles[0].SchedulerName == nil {
-		obj.Profiles[0].SchedulerName = pointer.StringPtr(api.DefaultSchedulerName)
```

Comment:
@hprateek43 did you mean to leave it as v1.DefaultSchedulerName? (k8s.io/kubernetes/pkg/apis/core.DefaultSchedulerName) is undefined.

Reply:
that was removed recently